### PR TITLE
Use selector from RC to delete PODs.

### PR DESCRIPTION
**Pull request by kadel**
Fixes: #484

### CODE REVIEW

File: atomicapp/providers/openshift.py [hunk 1]
```diff
@@ -257,7 +257,9 @@ def undeploy(self):
             # if there is ReplicationController we need delete all
             # Pods that were created by this RC
```
Comment (dustymabe) [inline atomicapp/providers/openshift.py]: Add to this comment:

```
# If this is a ReplicationController we need to delete all
# Pods that were created by this RC. Find the pods that 
# belong to this RC by querying for all pods and filtering 
# based on the selector used in the RC.
```

**Comment by dustymabe** [comment]
LGTM. Once you update the comment we can merge merge!

**Comment by kadel** [comment]
updated. Thank you @dustymabe

File: atomicapp/providers/openshift.py [hunk 2]
```diff
@@ -229,8 +229,10 @@ def undeploy(self):
 
             logger.info("Undeploying artifact name=%s kind=%s" % (name, kind))
 
-            # if there is DeploymentConfig we also need to delete all
-            # ReplicationControllers that were created by this DC
+            # If this is a DeploymentConfig we need to delete all
+            # ReplicationControllers that were created by this DC. Find the RC
+            # that belong to this DC by querying for all RC and filtering based
+            # on atomaticaly created label openshift.io/deployment-config.name
```
Comment (dustymabe) [inline atomicapp/providers/openshift.py]: s/atomaticaly/automatically

**Comment by dustymabe** [comment]
@cdrage, quick review please

**Comment by cdrage** [comment]
@kadel @dustymabe 
Tests pass from your PR and works well (testing manually too)!

**Pull request merged**